### PR TITLE
Ruby on Rails installation

### PR DESCRIPTION
**Pull request by diabolo**
- simplify explanation for using cucumber-rails see line 185 in my diff
- tidy formatting

### CODE REVIEW

File: content/installation.md [hunk 1]
```diff
@@ -173,17 +173,19 @@ Then, initialize a `features/` directory:
...
-[Bundler](http://gembundler.com/) is a utility gem that manages project specific RubyGem dependencies such that each
-Rails project is unaffected by the gem requirements of another.
-If you use Bundler to support multiple gem versions in multiple Rails projects on a single development host then you must
-run Cucumber using the preface `bundle exec`. For example: `bundle exec cucumber -pnone features`.
+[Bundler](http://gembundler.com/) is a utility gem that manages project
+specific RubyGem dependencies such that each Rails project is unaffected by
+the gem requirements of another.  If you use Bundler to support multiple gem
+versions in multiple Rails projects on a single development host then you must
+run Cucumber using the preface `bundle exec`. For example: `bundle exec
+cucumber -pnone features`.
```
Comment (luke-hill) [inline content/installation.md]: Do we want to advertise using `-pnone` here, which might be misinterpreted?

**Comment by aslakhellesoy** [comment]
I've split up this big file into several smaller ones. Could you please update the PR to modify `contents/installation/ruby.md` instead?

Sorry about the hassle.

**Comment by aslakhellesoy** [comment]
Hi @diabolo,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Comment by mlvandijk** [comment]
Updated this PR, so it could be merged. Thanks for your help @diabolo!

**Pull request merged**